Lib/dbm/sqlite3.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sqlite3
+from contextlib import suppress, closing
+from collections.abc import MutableMapping
+
+BUILD_TABLE = """
+  CREATE TABLE IF NOT EXISTS Dict (
+    key TEXT NOT NULL,
+    value BLOB NOT NULL,
+    PRIMARY KEY (key)
```

Comment:
IMO, we should let SQLite manage the primary key through the implicit `rowid`.

```suggestion
    key TEXT UNIQUE NOT NULL,
    value BLOB NOT NULL
```

Reply:
cc. @corona10

Reply:
- https://www.sqlite.org/lang_createtable.html

Reply:
I'll add some tests for key uniqueness and validity of the generated database.

Reply:
Ah, there is already generic tests for key uniqueness in `test_dbm.py`; we're covered.